client/landing/gutenboarding/components/domain-picker/button.tsx

```diff
@@ -27,11 +27,11 @@ const DomainPickerButton: FunctionComponent< Props > = ( {
 	queryParameters,
 	...buttonProps
 } ) => {
-	const buttonRef = createRef();
+	const buttonRef = createRef< HTMLButtonElement >();
 
 	const [ isDomainPopoverVisible, setDomainPopoverVisibility ] = useState( false );
 
-	const handleClose = ( e?: FocusEvent ) => {
+	const handleClose = ( e?: React.FocusEvent ) => {
```

Comment:
Alternatively, can't we just drop our [`onFocusOutside`](https://github.com/Automattic/wp-calypso/pull/38093/files#diff-d110ec85e6d318fd110d71ccd37806cbR64) from `Popover`?

Asking b/c https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ff69d22d5d124148da0a034711c573cf044f6a82/types/wordpress__components/popover/index.d.ts#L82

Reply:
That was required to handled when the click outside is _on the toggle that closes_ the popover. That would trigger the tuggle twice and result in the popover not being closed.